app/src/main/java/org/simple/clinic/medicalhistory/newentry/NewMedicalHistoryEffectHandler.kt

```diff
@@ -90,6 +91,18 @@ class NewMedicalHistoryEffectHandler @AssistedInject constructor(
     }
   }
 
+  private fun setupUiForDiabetesManagement(diabetesManagementEnabled: Boolean) {
+    uiActions.apply {
+      if (diabetesManagementEnabled) {
+        showDiagnosisView()
```

Comment:
Should we have these as part of UI renderer as well instead of effect handler?

Reply:
Good point, these should be in the UI renderer. I will make this change.

Reply:
Done.